### DISCUSSION TITLE
Audacious GTK/Qt Winamp interface unable to resize main window or equalizer

### DISCUSSION THREAD

**Discussion by BRTPOB** [audacious-media-player/audacious#1490] [Q&A] [answered]
This spawned out of #105, but using either GTK or Qt interfaces with the Winamp Classic interface, on Pop!_OS 22.04 (Ubuntu 22.04) the main window and equalizer are unable to be resized. The windows can be moved just fine, however.

The playlist window can be moved and resized.

Running 4.4.1 from the UbuntuHandbook PPA.

Using the ctrl+d shortcut does make the window size double.

**Comment by mschwendt** [comment, marked as answer]
It is by design that only the playlist window can be resized via its element in the bottom right corner. There is no such feature for the main window and the equalizer.

**Reply by radioactiveman** [reply]
Absolutely correct. And aside from the double size functionality resizing those windows was neither supported by XMMS or even the original Winamp 2.
